Codigo/MyScanner-OnlyDigital/DataAdcquisition.pas

```diff
@@ -134,7 +134,7 @@   TDataForm = class(TForm)
   SupraSPI_Hdl:Dword;
   simulating: Boolean;
   simulatedDac: array[0..7] of Integer;
-  Buffer:String[50]; //En ppio. hay espacio de sobra con esta cantidad
+  Buffer: AnsiString[50]; //En ppio. hay espacio de sobra con esta cantidad
```

Comment:
keep as string[50], as it is a statically defined short string rather than a UTF string